src/realm/util/client_websocket.hpp

```diff
@@ -16,14 +16,14 @@ namespace realm::util::websocket {
 using port_type = sync::port_type;
 
 // TODO figure out what belongs on config and what belongs on endpoint.
-struct EZConfig {
+struct SocketConfig {
```

Comment:
Seems like this is more of the SocketFactoryConfig than the SocketConfig. The SocketConfig is just the Endpoint.

Reply:
+1